Xamarin.Forms.Platform.Android/Resources/values/styles.xml

```diff
@@ -7,4 +7,28 @@
   <style name="NestedScrollBarStyle" android:id="@+id/nestedScrollViewStyle">
     <item name="android:scrollbars">vertical|horizontal</item>
   </style>
-</resources>
+    <style name="MainTheme" parent="MainTheme.Base">
+    </style>
+    <!-- Base theme applied no matter what API -->
+    <style name="MainTheme.Base" parent="Theme.AppCompat.DayNight.DarkActionBar">
```

Comment:
Will this theme work for projects targeting MonoAndroid9.0?

Reply:
Yes, it's built on top of the regular AppCompat themes so it will just fallback to the ones available there. Just took the NuGets and verified that it in fact does

![image](https://user-images.githubusercontent.com/939291/77417070-23831700-6dc5-11ea-8c37-fbd46ba78470.png)